src/client/datascience/jupyter/kernels/kernel.ts

```diff
@@ -248,4 +251,16 @@ export class Kernel implements IKernel {
         }
         await this.notebook.waitForIdle(this.launchTimeout);
     }
+
+    private disableJedi() {
+        if (this.uri.fsPath && isPythonKernelConnection(this.metadata)) {
```

Comment:
```suggestion
        if (isPythonKernelConnection(this.metadata)) {
```

Coz `this.uri` is never null